official/transformer/model/beam_search_test.py

```diff
@@ -43,22 +47,22 @@ def test_shape_list(self):
 
   def test_get_shape_keep_last_dim(self):
     y = tf.constant(4.0)
-    x = tf.ones([7, tf.to_int32(tf.sqrt(y)), 2, 5])
+    x = tf.ones([7, tf.compat.v1.to_int32(tf.sqrt(y)), 2, 5])
```

Comment:
I think you can replace `tf.compat.v1.to_int32(...)` with `tf.cast(tf.sqrt(y), tf.int32)`

Reply:
Yeah I thought about doing this right and was lazy and maybe not as sure of myself as I wanted to be.  I'll make the change, thank you for the tip.